Dockerfile-worker

```diff
@@ -1,8 +1,18 @@
 FROM python:3.10-slim
 
-# install gfortran via apt and clean cache
+# install libraries required to run simulators via apt and clean cache
 # skipcq: DOK-DL3008
-RUN apt-get update && apt-get install -y --no-install-recommends gfortran && rm -rf /var/lib/apt/lists/*
+RUN apt-get update && apt-get install -y --no-install-recommends \
+    libexpat1-dev \
+    libgl1-mesa-dev \
+    libglu1-mesa-dev \
+    libxt-dev \
+    xorg-dev \
+    build-essential \
+    libharfbuzz-dev \
+    gfortran \
+    && \
+    rm -rf /var/lib/apt/lists/*
```

Comment:
what is the impact of installation of these packages on the docker image size ?

Reply:
image size with these packages (but without TOPAS): 20.3MB (virtual 732MB)
vs size without these packages: 483MB (virtual 1.19GB)